TravelApp.WPF/flight_oder.xaml

```diff
@@ -0,0 +1,43 @@
+﻿<UserControl x:Class="TravelApp.WPF.flight_oder"
```

Comment:
The component should be in folder "Components". This folder is in TravelApp.WPF/Components